sds_data_manager/lambda_code/SDSCode/batch_starter.py

```diff
@@ -280,81 +283,52 @@ def prepare_data(filename, upstream_dependencies):
         Data to submit to batch job.
 
     """
-    components = extract_components(filename)
-
-    instrument = components["instrument"]
-    level = components["datalevel"]
-    start_date = components["startdate"]
-
-    format_start_date = datetime.strptime(start_date, "%Y%m%d")
-
-    # Format year and month from the datetime object
-    year = format_start_date.strftime("%Y")
-    month = format_start_date.strftime("%m")
-
-    # Base S3 path
-    s3_base_path = f"imap/{instrument}/{level}/{year}/{month}/"
-
-    # Prepare the final command
-    # Pre-construct parts of the string
+    # Prepare batch job command
     # NOTE: Batch job expects command like this:
     # "Command": [
-    #     "--instrument",
-    #     "swe",
-    #     "--level",
-    #     "l1b",
-    #     "--file_path",
-    #     "imap/swe/l1b/2023/09/imap_swe_l1b_lveng-hk_20230927_20230927_v01-00.cdf",
-    #     "--dependency",
-    #     "[{'instrument': 'swe', 'level': 'l0', 'version': 'v00-01'}]"
-    #   ]
+    #     "--instrument", "mag",
+    #     "--level", "l1a",
+    #     "--start-date", "20231212",
+    #     "--end-date", "20231212",
+    #     "--version", "v00-01",
+    #     "--dependency", """[
+    #         {
+    #             'instrument': 'swe',
+    #             'data_level': 'l0',
+    #             'descriptor': 'lveng-hk',
+    #             'start_date': '20231212',
+    #             'end_date': '20231212',
+    #             'version': 'v01-00',
+    #         },
+    #         {
+    #             'instrument': 'mag',
+    #             'data_level': 'l0',
```

Comment:
Small change: use either 'level' or 'data_level'.

Reply:
I was concerned about that too. batch job command(aka imap_cli) uses `level` and all the other places uses`data_level`. @maxinelasp can you update `imap_cli` to use `data_level` as well and I can change it here.

Reply:
I think I already made that change yesterday? Let me double check

Reply:
I changed dependency to `data_level` but not the CLI itself. Now, everything should be data_level. Thanks for catching that, it's great to get all these names sorted out now!

Reply:
Yes. Thank you both so much!

Reply:
I updated places where I can from `level` to `data_level`. I didn't change it in places where we use `level` when we build dependency. We will need to update that in upcoming work. so I left it as it is for now.

Reply:
I decided to change all afterwards :) and tested in AWS. now the batch job command looks like this 
```
--instrument swe --data_level l1a --start-date 20230927 --end-date 20230927 --version v03-00 --dependency [{'instrument': 'swe', 'data_level': 'l0', 'version': 'v03-00'}] --use-remote
```